website/src/assets/pages/vis6-maps-of-health/vis.vl.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+  "data": {"url": "http://localhost:33284/assets/generated/visualization6/data1.csv",
+    "format": {
+      "type":"csv",
+      "parse": {
+        "PERIOD":"date",
+        "DOD":"date",
+        "AGE":"number",
+        "COUNT": "number",
+        "RANK": "number",
+        "Encounter_Type": "string"
+      }
+    }
+  },
+  "transform": [
+    {
+      "calculate": "round(12 * (datum.DOD - datum.PERIOD) / 3.154e+10)",
+      "as": "TIME_BEFORE_DEATH"
+    },
+    {
+      "filter": "datum.TIME_BEFORE_DEATH >= 0"
+    }
+  ],
+  "spacing": {"row": -20},
+  "facet": {
+    "row": {
+      "field": "RANK",
+      "type": "nominal",
+      "bin": false,
+      "header": null
+    }
+  },
+  "spec": {
+    "title": " ",
+    "width": 700,
+    "height": 20,
+    "view": {"stroke": null},
+    "mark": { "type": "area", "line": {"size": 0.5} },
+    "encoding": {
+      "x": {
+        "field": "TIME_BEFORE_DEATH",
+        "type": "quantitative",
+        "title": "Months Before Death",
+        "axis": { "orient": "top" },
+        "scale": {
+          "domainMin": 0,
+          "range": ["width", 0]
+        }
+      },
+      "y": {
+        "field": "COUNT",
+        "type": "quantitative",
+        "scale": {"zero": true},
+        "stack": "zero",
+        "axis": {
+          "title": null,
+          "labels": false,
+          "ticks": false,
+          "grid": false
+        }
+      },
+      "color": {
+        "field": "Encounter_Type",
```

Comment:
Try setting axis.title to "Encounter Type" then you can revert to the field being more normal (ENCOUNTER_TYPE).